app/screens/user_profile/index.js

```diff
@@ -14,9 +15,12 @@ import {loadBot} from '@mm-redux/actions/bots';
 import {getRemoteClusterInfo} from '@mm-redux/actions/remote_cluster';
 import {getBotAccounts} from '@mm-redux/selectors/entities/bots';
 import {getCurrentUserId} from '@mm-redux/selectors/entities/users';
+import {makeGetCustomStatus} from '@selectors/custom_status';
 
 import UserProfile from './user_profile';
 
+const getCustomStatus = makeGetCustomStatus();
```

Comment:
I recently found that to avoid unnecessary re-renders this needs to be used in a mapStateToProps factory like

```js
function mapStateToProps() {
    const getCustomStatus = makeGetCustomStatus();
    return (state, ownProps) => {
        ....
    };
}
```